app/src/main/java/com/alexbar/layoutbasic/screen/confirm_payment/screens/ConfirmPaymentScreen.kt

```diff
@@ -0,0 +1,161 @@
+package com.alexbar.layoutbasic.screen.confirm_payment.screens
+
+import android.annotation.SuppressLint
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.Button
+import androidx.compose.material3.SnackbarDuration
+import androidx.compose.material3.SnackbarHost
+import androidx.compose.material3.SnackbarHostState
+import androidx.compose.material3.SnackbarResult
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableIntStateOf
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.rememberCoroutineScope
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.constraintlayout.compose.ConstraintLayout
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.AddCreditCardDialog
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.CreditCardInfo
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.CreditCardList
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.EmptyCreditCards
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.HeaderConfirmPayment
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.SegmentedControlConfirmPayment
+import com.alexbar.layoutbasic.screen.confirm_payment.widgets.UserBalance
+import com.alexbar.layoutbasic.ui.theme.Dimens.dimen_16_dp
+import com.alexbar.layoutbasic.utils.ConfirmPayment.add_credit_cards_button
+import com.alexbar.layoutbasic.utils.ConfirmPayment.checkout_button
+import com.alexbar.layoutbasic.utils.ConfirmPayment.current_user_balance
+import com.alexbar.layoutbasic.utils.ConfirmPayment.no_credit_cards_message
+import com.alexbar.layoutbasic.utils.ConfirmPayment.payment_method_credit_card
+import com.alexbar.layoutbasic.utils.ConfirmPayment.payment_method_title
+import com.alexbar.layoutbasic.utils.ConfirmPayment.payment_method_user_balance
+import com.alexbar.layoutbasic.utils.ConfirmPayment.total_amount
+import com.alexbar.layoutbasic.utils.ConfirmPayment.welcome_message
+import kotlinx.coroutines.launch
+
+@SuppressLint("MutableCollectionMutableState")
+@Composable
+fun ConfirmPaymentScreen() {
+    val scope = rememberCoroutineScope()
+    val snackBarHostState = remember { SnackbarHostState() }
+    var selectedIndex by remember { mutableIntStateOf(0) }
```

Comment:
var selectedIndex by remember { mutableStateOf(0) }

 why ? :)
 Instead of using mutableIntStateOf, consider using mutableStateOf for selectedIndex if the index is used for UI-related purposes. This can make your **code consistent** and **easier to understand**.